Documentation/devicetree/bindings/media/i2c/irs1125.txt

```diff
@@ -0,0 +1,49 @@
+* Infineon irs1125 time of flight sensor
+
+The Infineon irs1125 is a time of flight digital image sensor with
+an active array size of 352H x 286V. It is programmable through I2C
+interface. The I2C address defaults to 0x3D, but can be set to 0x3C or
+none standard mipi address 0x41.
```

Comment:
non standard.
What do you mean by a mipi address?

Reply:
I mean mipi csi-2. I changed this description a bit. Mipi CSI-2 suggests these I2C addresses:

6.2 CCI Slave Addresses

For camera modules having only raw Bayer output the 7-bit slave address should be 011011Xb, where X =
0 or 1. For all other camera modules the 7-bit slave address should be 011110Xb

Reply:
Wow, never noticed that bit in the CSI spec, and rarely seen it followed! (Having said that, OV5647 is on 0x36, which is 011011Xb)

TBH I'd drop reference to non-standard MIPI addresses. Specify the I2C addresses that the module can be configured to, and leave it at that. 
"The I2C address will be one of 0x3C, 0x3D, or 0x41, selected by <some mechanism>."
You'll get more detailed feedback on the DT bindings should you try and upstream them.

Reply:
Okay, maybe we have another problem here. The default address after power on is 0x3D, you can than change the address to be either 0x3C, 0x3D or 0x41 by writing to a register. Would it be valid to support only 0x3D or how should that one be modeled?

Reply:
Joys! It's one of them.
I seem to recall one other sensor driver does support dynamic changing of the I2C address because people were wanting to use it for stereoscopic capture, but it gets very messy in doing so.

Yes, supporting only 0x3D in the driver is valid.  DT describes what the hardware can potentially do. Drivers don't have to implement all of that (eg DT describes a clock source, but the driver can reject it should the clock not be the one it wanted).

"The I2C address defaults to 0x3D, but can be reconfigured to address 0x3C or 0x41 via I2C commands"